openbas-framework/src/main/java/io/openbas/expectation/ExpectationBuilderService.java

```diff
@@ -12,14 +12,13 @@ public class ExpectationBuilderService {
 
   private final ExpectationPropertiesConfig expectationPropertiesConfig;
 
-  public static Double DEFAULT_TECHNICAL_EXPECTATION_SCORE = 100.0;
-  public static Double DEFAULT_HUMAN_EXPECTATION_SCORE = 0.0;
+  public static Double DEFAULT_EXPECTATION_SCORE = 100.0;
 
   public Expectation buildPreventionExpectation() {
     Expectation preventionExpectation = new Expectation();
     preventionExpectation.setType(PREVENTION);
-    preventionExpectation.setName("Expect inject to be prevented");
-    preventionExpectation.setScore(DEFAULT_TECHNICAL_EXPECTATION_SCORE);
+    preventionExpectation.setName("Prevention");
```

Comment:
nitpick: use a constant? Also, how do we manage the translation on the frontside of this default name?